utils/agljobtemplate.py

```diff
@@ -62,7 +62,9 @@ def render_job(self, url, machine, job_name = "AGL-short-smoke", priority = "med
         job['priority'] = priority
         job['urlbase'] = url
         job['test_templates'] =  test_templates
-        job['rootfs_type'] = rfs_type
+
+        if rfs_type is not None:
```

Comment:
You could just use `if rfs_type`

Reply:
Indeed, but the rest on python scripts are written that way
I have no preference ... I suppose there is 'python recommended way' for this
Either, we should be consistent about it ... so maybe another patch later on ?

Reply:
Ok I can do a patch following pep8 coding style later on :)